prov/efa/src/efa_hmem.c

```diff
@@ -283,9 +283,10 @@ ssize_t efa_copy_from_hmem_iov(void **desc, char *buff, int buff_size,
  * @return  number of bytes copied on success, or a negative error code
  */
 ssize_t efa_copy_to_hmem_iov(void **desc, struct iovec *hmem_iov,
-                             int iov_count, char *buff, int buff_size)
+                             size_t iov_count, char *buff, size_t buff_size)
 {
-	int i, ret, bytes_remaining = buff_size, size;
```

Comment:
how can it work before?

Reply:
I believe it's ultimately implementation-defined, but `int` and `size_t` are almost certainly the sizes you'd expect for our use-case. This PR simply minimizes potential risks wrt data type mismatch/integer promotion.